python/ray/serve/handle.py

```diff
@@ -1,9 +1,26 @@
-from typing import Optional, Dict, Any, Union
+import asyncio
+import threading
+from typing import Any, Coroutine, Dict, Optional, Union
 
+import ray
 from ray.serve.context import TaskContext
-from ray.serve.router import RequestMetadata
+from ray.serve.router import RequestMetadata, Router
 from ray.serve.utils import get_random_letters
 
+global_async_loop = None
+
+
+def create_or_get_async_loop_in_thread():
+    global global_async_loop
+    if global_async_loop is None:
```

Comment:
fyi the resetting the default event loop policy is gone (probably some false positive during testing and debugging)